exec/java-exec/src/test/java/org/apache/drill/exec/work/batch/FileTest.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) throws IOException {
     FSDataOutputStream out = fs.create(path);
     byte[] s = "hello world".getBytes();
     out.write(s);
-    out.sync();
+    out.hsync();
```

Comment:
I see in hadoop-common that sync is replaced with `Syncable#hflush()`. Do we need use `hflush` instead of `hsync`?